### PR TITLE
style: indentation and functions

### PR DESCRIPTION
**Pull request by linuxbandit**

### CODE REVIEW

File: scripts/myaegeestatus.js [hunk 1]
```diff
@@ -31,35 +31,46 @@
 //   https://github.com/linuxbandit
 
 module.exports = (robot) => {
+
+  function alwaysThread(res) {
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:35]: I suggest to move it to other file (like `helper.js`, probably not in `scripts`/ folder), so we can reuse it in all files

Comment (linuxbandit) [reply scripts/myaegeestatus.js:35]: I was thinking about it, then I was afraid hubot would mess up. I will try in local tho

Comment (serge1peshcoff) [reply scripts/myaegeestatus.js:35]: that's why I suggest to have it not in `scripts/` folder

Comment (linuxbandit) [reply scripts/myaegeestatus.js:35]: yeah doesn't work

Comment (serge1peshcoff) [reply scripts/myaegeestatus.js:35]: what does it say though?

**Comment by linuxbandit** [comment]
@serge1peshcoff did you have the chance to try yourself?

**Comment by linuxbandit** [comment]
LOL I didn't think about putting it outside of `scripts/` -.- silly me

**Pull request merged**